vspace/src/main/java/edu/asu/diging/vspace/core/services/IExhibitionAboutPageManager.java

```diff
@@ -17,10 +20,13 @@ public interface IExhibitionAboutPageManager {
     
     /**
      * This method stores Exhibition About page information and returns the {@link ExhibitionAboutPage} which is being stored
-     * @param {@link ExhibitionAboutPage} This object contains Exhibition about page values to be stored.
+     * @param aboutPageForm
      * @return {@link ExhibitionAboutPage} Returns the object after being stored. Object will now have an ID.
      */
-    ExhibitionAboutPage store(ExhibitionAboutPage exhibitionAboutPage);
+    ExhibitionAboutPage store(AboutPageForm aboutPageForm);
 
     ExhibitionAboutPage getExhibitionAboutPage();
+
+    AboutPageForm createAboutPageForm();
```

Comment:
forms should not be created by the manager since they are just for the front end